spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/config/EurekaConfigServerBootstrapConfiguration.java

```diff
@@ -69,8 +74,6 @@ public EurekaClientConfigBean eurekaClientConfigBean() {
 
 	@Bean
 	@ConditionalOnMissingBean(EurekaHttpClient.class)
-	@ConditionalOnProperty(prefix = "eureka.client", name = "webclient.enabled", matchIfMissing = true,
```

Comment:
From my analysis, it seems that previously, `resttemplate` was set as default even if `webclient` was disabled. However, after that, I felt that the above setting was a bit ambiguous because there are two options even if `webclient` is disabled. What do you think? 🤔

Reply:
We want to preserve the default behaviour and only add opt-in `RestClient` support at this point. We may want to change it in a `major` release at this point. If `webClient` is not enabled, we want to use `RestTemplate`, and there should be a similar `@Conditional` added for `restClient` as well.

Reply:
Sorry for the late reply.

If so, I will revert the annotation and change it to only be enabled if we want to use `restClient`.